### PR TITLE
Merge Find-GcsObject into Get-GcsObject; Fix error message for Remove-GcsObject when object not found

### PR DESCRIPTION
**Pull request by quoctruong**
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/377)

### CODE REVIEW

File: Google.PowerShell/Storage/GcsObject.cs [hunk 1]
```diff
@@ -398,17 +398,51 @@ private static string ConvertLocalToGcsFolderPath(string localFilePath)
...
     /// </para>
     /// <para type="description">
-    /// Returns the give Storage object's metadata.
+    /// If no parameter besides -Bucket is provided, all objects in the bucket are returned.
+    /// If a given prefix string is provided, returns all Cloud Storage objects identified
+    /// by the prefix string.
+    /// </para>
+    /// <para type="description">
+    /// An optional delimiter may be provided. If used, will return results in a
+    /// directory-like mode, delimited by the given string. e.g. with objects "1,
```
Comment (iantalarico) [inline Google.PowerShell/Storage/GcsObject.cs]: The description and example are a little hard to follow.

Is this well documented for GCS?

Can you add a more detailed example?

File: Google.PowerShell/Storage/GcsObject.cs [hunk 2]
```diff
@@ -419,37 +453,90 @@ public class GetGcsObjectCmdlet : GcsCmdlet
...
+                // When used with WriteObject, expand the IEnumerable rather than
+                // returning the IEnumerable itself. IEnumerable<T> vs. IEnumerable<IEnumerable<T>>.
+                const bool enumerateCollection = true;
+
+                // First page.
+                Objects gcsObjects = listReq.Execute();
+                WriteObject(gcsObjects.Items, enumerateCollection);
+
+                // Keep paging through results as necessary.
+                while (!Stopping && gcsObjects.NextPageToken != null)
```
Comment (iantalarico) [inline Google.PowerShell/Storage/GcsObject.cs]: I'm guessing this is a powershell thing I'm missing but where is "Stopping" coming from?

Comment (quoctruong) [inline Google.PowerShell/Storage/GcsObject.cs]: It's a property of the PowerShell cmdlet class. It is true if the cmdlet execution stopped (like if user press Ctrl + C).

File: Google.PowerShell/Storage/GcsObject.cs (hunk 1 above)
Comment (quoctruong) [inline Google.PowerShell/Storage/GcsObject.cs]: Is this description better? "Returns results in a directory-like mode. items will contain only objects whose names, aside from the prefix, do not contain delimiter. Objects whose names, aside from the prefix, contain delimiter will have their name, truncated after the delimiter, returned in prefixes. Duplicate prefixes are omitted." This is from [here](https://cloud.google.com/storage/docs/json_api/v1/objects/list).

Comment (iantalarico) [inline Google.PowerShell/Storage/GcsObject.cs]: Ah yea that is much more clear, thanks :)

**Comment by quoctruong** [comment]
Thanks!
